src/components/modals/ErrorModal/ErrorModal.js

```diff
@@ -0,0 +1,81 @@
+import React from 'react'
+import styled from 'styled-components/macro'
+import PropTypes from 'prop-types'
+
+ErrorModal.propTypes = {
+  handleErrorOK: PropTypes.func.isRequired,
+}
+
+function ErrorModal({ handleErrorOK }) {
+  return (
+    <ErrorWrapper>
+      <Modal>
+        <ErrorTitle>Ein Fehler ist beim Laden aufgetreten!</ErrorTitle>
+        <InfoMessage>
+          Die App wird zurück gesetzt. Alle Daten sind verloren!
+        </InfoMessage>
+        <Action onClick={handleErrorOK}>OK</Action>
+      </Modal>
+    </ErrorWrapper>
+  )
+}
+
+export default ErrorModal
+
+const ErrorWrapper = styled.div`
+  background-color: rgba(0, 0, 0, 0.7);
+  bottom: 0;
+  height: 100vh;
+  left: 0;
+  position: absolute;
+  right: 0;
+  top: 0;
+  width: 100wh;
+  z-index: 100;
+`
+
+const Modal = styled.section`
+  background-color: #f3f3f3;
+  border-radius: 10px;
+  border: 1px solid #ccc;
+  display: flex;
+  flex-direction: column;
+  height: 18em;
+  justify-content: space-between;
+  left: 50%;
+  margin-left: -10em;
+  margin-top: -9em;
+  padding: 0;
+  position: fixed;
+  top: 50%;
+  width: 20em;
+`
+
+const ErrorTitle = styled.h2`
+  color: red;
+  font-size: 20px;
+  padding: 10px 0;
+  text-align: center;
+`
+
+const InfoMessage = styled.p`
+  font-size: 18px;
+  padding: 0 10px;
+  text-align: center;
+`
+
+const Action = styled.button`
```

Comment:
The button in the AlertModal has similar styles. Maybe it makes sense to have a shared Button component. If you need to overwrite some styles you can still use styled(Button).